content/blog/2016/2016-09-30-jenkins-world-2016-wrap-up-community.adoc

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title: "Jenkins World 2016 Wrap-up - Community"
```

Comment:
Here are some ideas for catchier titles:

* "Jenkins World 2016, that's a wrap"
* "Jenkins World 2016 Wrap-up - Looking Forward"
* "Jenkins World 2016 Wrapped Up"